src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return firstLeg * secondLeg / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: RightTriangle, area: " + calculateArea()
+                + " sq. units, firstLeg: " + firstLeg
+                + " units, secondLeg: " + secondLeg
+                + " units, color: " + color);
```

Comment:
Use only eng in messages/code: The message in the draw method should be in English. The term 'sq. units' may be confusing as it's not a common abbreviation for 'square units'. Consider using the full term for clarity.